pages/news/asteroid-hosttools.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Introducing asteroid-hosttools
+layout: news
+posted: 2023-05-01
```

Comment:
I didn't know you were a time traveler!

Reply:
Ow, forgot to draft this PR, hehe.
My plan was to merge it in may so we stretch the few we got monthly?

But i am easy to convince otherwise in case others have plans for more articles soonish?

Reply:
Set to release at tomorrow 2023-04-24, now that i know we are awaiting more posts even :tada: